decorator/timer.py

```diff
@@ -0,0 +1,23 @@
+import time
+
+def decorator(function):
+    def wew(*args, **kwargs):
+        start = time.time()
+        x = function(*args, **kwargs)
+        end = time.time()
+        print(function.__name__, "execution time", end - start, "sekund.")
```

Comment:
To zasadniczo jest w porządku, natomiast od Pythona 3.6 mamy f-stringi, które są chyba najczytelniejszą formą formatowania stringów, więc polecałbym takie coś zrobić w ten właśnie sposób - oczywiście jest to kwestia subiektywna, ale byłoby to też spójne z resztą tego repo.

Reply:
A drugi komentarz do tej linijki jest taki, że raczej powinniśmy utrzymywać wszystko po angielsku w kodzie.